### ISSUE TITLE
Bullet lists transformed to numbered lists

### ISSUE THREAD

**Issue by kdahlsp** [SPFxAppDev/docxmerger#2]
Noticed that when merging two files containing bullet lists the output is transformed to numbered lists. 
Also that the numbering seem to continue cross the document, i.e. not reset back to 1 per list instance.
Do you know if there is a fix for this?

**Comment by SPFxAppDev** [comment]
Hi, thanks for your report. I'll have to take a closer look. The logic is that of the original repository, so it's probably the same there. I'll let you know as soon as I've had a look and possibly have a solution. At the moment, however, I have a lot to do and don't know when I'll get around to it...

**Comment by kdahlsp** [comment]
Hi and thanks for your reply. I suspected so, will try to verify with the original but had issues with it in my SPFx project which is where I found yours. Thanks anyhow to your contribution on the module.

**Comment by kdahlsp** [comment]
And yes, the issue is in the original repo docx-merger.

**Comment by SPFxAppDev** [comment]
Hi @kdahlsp,

my use case and the reason I created my own npm package was similar to yours. I needed something like merging in an SPFx solution. Unfortunately, there is no alternative package/library. So I customized the original. But for my requirements I only needed “basics”. It was a very simple Word file (for certification creation). So I'm sure there are more bugs in the package (the original package was never finalized). I have now looked at your problem and I think I have fixed it. Could you test it please? It worked for me. It is included in version 1.0.3.

**Comment by kdahlsp** [comment]
Hi, thanks for your swift commit. 

I would say improved but still some issues. Will have a look as well and see if I can find anything. But as I learn a little bit about how the relation work between the document.xml and the numbering.xml it seem prone to introduce conflicts when merging from multiple sources. I guess this is why when testing it can look almost identical but when introducing more documents, with their own styling/numberings, it goes off.

**Comment by SPFxAppDev** [comment]
That is correct. There are so many variations that you probably can't even take them into account. Yesterday I worked with a standard bullet and standard numbering list and there were no problems. Even that took me almost a day (in case you're interested: it was due to the order of the XML elements). 

What also doesn't work (which is also logical), for example, is if you have several documents that each use their own footer/header, then there are also problems.

**Comment by kdahlsp** [comment]
Yeah I saw that as well. What I might investigate is to supply default template files for numbering.xml and maybe styles.xml, and override the document content to fit those. Guess another option would be to try to rewrite each document using docx module. Thanks anyway!

**Comment by SPFxAppDev** [comment]
Styles.xml should be merged (as it was implemented in the original repository). But I haven't tested it yet. Maybe you can attach your files (or similar files if the content is private) and I can take a look. To be honest, I'm not an expert on Word XML.

**Comment by kdahlsp** [comment]
No no that´s all right, thank you. Yeah I´m thinking of excluding numbering and styles and see if its possible to supply a more generic as base, and remap the document nodes to those.
